python/examples/test-py-arch.py

```diff
@@ -0,0 +1,92 @@
+# Example Python Arch plugin written in Python
+# ============================================
+#
+#  -- pancake @ nopcode.org
+#
+# The r2lang.plugin function exposes a way to register new plugins
+# into the RCore instance. This API is only available from RLang.
+# You must call with with '#!python test.py' or 'r2 -i test.py ..'
+
+import r2lang
+from r2lang import R
+
+def pyarch(a):
+	def regs():
+		return "=PC	pc\n" + \
+		"=SP	sp\n" + \
+		"=A0	r0\n" + \
+		"gpr	r0	.32	0	0\n" + \
+		"gpr	r1	.32	4	0\n" + \
+		"gpr	r2	.32	8	0\n" + \
+		"gpr	r3	.32	12	0\n" + \
+		"gpr	r4	.32	16	0\n" + \
+		"gpr	r5	.32	20	0\n" + \
+		"gpr	sp	.32	24	0\n" + \
+		"gpr	pc	.32	28	0\n"
+
+	def info(query):
+		if query == R.R_ARCH_INFO_MINOP_SIZE:
```

Comment:
Maybe worth having a dict here and use if query in dict syntax instead of many ifs